docs/development/build.md

```diff
@@ -30,11 +30,25 @@ $GOPATH/src/github.com/google/cadvisor $ make test
 
 For integration tests, see the [integration testing](integration_testing.md) page.
 
+### Non-volatile Memory Support
+
+cAdvisor can be linked against [libipmctl](https://github.com/intel/ipmctl) library that allows to gather information about Intel® Optane™ DC Persistent memory. If you want to build cAdvisor with libipmctl support you must meet following requirements:
+* `libimpctl-devel` must be installed on build system.
+* `libimpctl` must be installed on all systems where cAdvisor is running.
+
+Detailed information about building `libipmctl` can be found in the project's [README](https://github.com/intel/ipmctl#build). Make sure to use the most up to date released version.
```

Comment:
Is it possible to indicated tested working version? To avoid a situation in the future that we don't know on which version of ipmctl it was possible to get data for cAdvisor.